src/native-stack/views/NativeStackView.tsx

```diff
@@ -107,6 +107,122 @@ const MaybeNestedStack = ({
   return content;
 };
 
+type RouteViewProps = {
+  route: Route<string>;
+  index: number;
+  navigation: NativeStackNavigationHelpers;
+  descriptors: NativeStackDescriptorMap;
+  navigatorKey: string;
+};
+
+function RouteView({
```

Comment:
what now is the purpose of this component? I remember previously when we discussed some implementation in the past there were some animated values being generated on a per route basis, but now it seems to no longer be the case?

Reply:
Yes, I'll revert it.